tests/tests_geomstats/test_geometric_median.py

```diff
@@ -0,0 +1,36 @@
+"""Methods for testing the Geometric Median Estimators."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.spd_matrices import SPDMatrices, SPDMetricAffine
+from geomstats.learning.geometric_median import WeiszfeldAlgorithm
+
+ROOT2 = gs.sqrt(2)
+
+
+class TestGeometricMedianEstimation(geomstats.tests.TestCase):
+    """Test of Geometric Meidan Estimators"""
+
+    def test_median_for_SPD_manifold(self):
```

Comment:
no uppercase in methods' names, as per international coding conventions in Python
--> test_median_for_spd_manifold